README.md

```diff
@@ -1,38 +1,19 @@
-Ontology management
-====
+# Ontology management
```

Comment:
you will need to resolve the conflicts first (rebase and resolve).
The readme in main has changed a lot since this PR was raised.
After this we can have a look what changed in this PR.